zeppelin-plugins/launcher/yarn/src/main/java/org/apache/zeppelin/interpreter/launcher/YarnRemoteInterpreterProcess.java

```diff
@@ -266,6 +270,30 @@ private ContainerLaunchContext setUpAMLaunchContext() throws IOException {
         addResource(fs, destPath, localResources, LocalResourceType.ARCHIVE, "hive_conf");
       }
     }
+
+    String yarnDistArchives = launchContext.getProperties().getProperty("zeppelin.yarn.dist.archives");
+    if (org.apache.commons.lang3.StringUtils.isNotBlank(yarnDistArchives)) {
```

Comment:
In the file, `org.apache.commons.lang3.StringUtils` is used 7 times, and `org.apache.hadoop.util.StringUtils` is used 1 time. I think it would be okay to use `org.apache.commons.lang3.StringUtils` in short form.

Reply:
Good point, fixed it now